run/GEOS/GEOSCHEMchem_ExtData.rc

```diff
@@ -395,6 +395,11 @@ QFED_C3H8_TP_SFC  'kg_m-2_s-1'     N    Y %y4-%m2-%d1T12:00:00   none      0.816
 QFED_C3H8_BO_SFC  'kg_m-2_s-1'     N    Y %y4-%m2-%d1T12:00:00   none      0.81633 biomass_xf     /discover/nobackup/projects/gmao/share/dao_ops/fvInput_nc3/PIESA/sfc/QFED/NRT/v2.5r1_0.1_deg/Y%y4/M%m2/qfed2.emis_c3h8.006.%y4%m2%d2.nc4
 QFED_C3H8_SV_SFC  'kg_m-2_s-1'     N    Y %y4-%m2-%d1T12:00:00   none      0.81633 biomass_sv     /discover/nobackup/projects/gmao/share/dao_ops/fvInput_nc3/PIESA/sfc/QFED/NRT/v2.5r1_0.1_deg/Y%y4/M%m2/qfed2.emis_c3h8.006.%y4%m2%d2.nc4
 QFED_C3H8_GL_SFC  'kg_m-2_s-1'     N    Y %y4-%m2-%d1T12:00:00   none      0.81633 biomass_gl     /discover/nobackup/projects/gmao/share/dao_ops/fvInput_nc3/PIESA/sfc/QFED/NRT/v2.5r1_0.1_deg/Y%y4/M%m2/qfed2.emis_c3h8.006.%y4%m2%d2.nc4
+QFED_PRPE_TF_SFC  'kg_m-2_s-1'     N    Y %y4-%m2-%d1T12:00:00   none      0.81633 biomass_tf     /discover/nobackup/projects/gmao/share/dao_ops/fvInput_nc3/PIESA/sfc/QFED/NRT/v2.5r1_0.1_deg/Y%y4/M%m2/qfed2.emis_c3h6.006.%y4%m2%d2.nc4
```

Comment:
Should these additional QFED fields (PRPE) be added to the standard model config as well?

Reply:
I don't know how many people are using QFED but it makes sense to include this in the standard model, in my opinion.